swap_meet/vendor.py

```diff
@@ -1,2 +1,80 @@
+from curses import wrapper
+from re import T
+from signal import set_wakeup_fd
+from pytest import Item
+from swap_meet.item import Item 
+
+
+
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory =  None):
+        if inventory == None:
+            inventory = list(())
+        self.item = Item()
+        self.inventory = inventory
+
+            
+    def add(self, item):
+        self.item = item 
+        self.inventory.append(item)
+        return item 
+
+    def remove(self, item):
+        if item in self.inventory:
+            self.inventory.remove(item)
+            return item 
+        else:
+            return False
+
+    def get_by_category(self, category):
+        category_list = []
+        for item in self.inventory:
+            if category == item.category:
+                category_list.append(item)
+        return category_list
+    
+    def swapping (self, vendor, my_item, their_item):
+        vendor.inventory.append(my_item)
+        vendor.inventory.remove(their_item)
+        self.inventory.append(their_item)
+        self.inventory.remove(my_item)
```

Comment:
you can use the methods you already created for add and remove. It would be like: `vendor.add(my_item)` and so on